src/EnrollWindowsHelloController.js

```diff
@@ -110,25 +126,75 @@ function (Okta, FormController, FormType, webauthn, Footer, WindowsHelloErrorMes
 
       formChildren: function () {
         var result = [];
+
         if (!webauthn.isAvailable()) {
           result.push(
             FormType.View(
-              { View: WindowsHelloErrorMessageView },
-              { selector: '.o-form-error-container' }
+              { View: new WindowsHelloErrorMessageView(
+                { message: Okta.loc('enroll.windowsHello.error.notWindows', 'login') }) },
+              { selector: '.o-form-error-container'}
             )
           );
         }
+        else {
+          result.push(FormType.View({
```

Comment:
Hmmm, are you sure about that? I can look into the code later, but I just tried the primaryAuth and resetPassword flows and they show the errors below the subtitle.